android/app/src/main/java/io/peggyjo/peggymeter/HistoryTextFragment.java

```diff
@@ -31,6 +38,37 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         dataController.addMoodListener(this);
         // Inflate the layout for this fragment
         View view = inflater.inflate(R.layout.fragment_history_text, container, false);
+        ListView table = view.findViewById(R.id.LOG_ENTRIES_CONTAINER);
+        table.setOnItemLongClickListener(new AdapterView.OnItemLongClickListener() {
+            @Override
+            public boolean onItemLongClick(AdapterView<?> adapterView, View view, int i, long l) {
```

Comment:
Instead of "i" and "l" can you use more descriptive names for better readability please?

Reply:
These references have been changed to int position and long id.